spec/fortnox/api/repositories/examples/save.rb

```diff
@@ -0,0 +1,61 @@
+# Assumes that attribute_hash_name holds a string without restrictions.
+shared_examples_for '.save' do |attribute_hash_name, required_attributes|
```

Comment:
Do this instead: `shared_examples_for '.save' do |attribute_hash_name, required_attributes = {}|`